ci-operator/step-registry/gcp/provision/private-dns-zone-in-host-project/gcp-provision-private-dns-zone-in-host-project-ref.yaml

```diff
@@ -0,0 +1,19 @@
+ref:
+  as: gcp-provision-private-dns-zone-in-host-project
+  from_image:
+    namespace: ocp
+    name: "4.12"
+    tag: upi-installer
+  grace_period: 10m
+  commands: gcp-provision-private-dns-zone-in-host-project-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: BASE_DOMAIN
+    documentation: The base domain.
+  - name: CLUSTER_PVTZ_PROJECT
```

Comment:
We can get `hostProject` from `xpn_project_setting.json`, so sounds we do not need this var.

Reply:
Do you mean, to use the var in the workflow, the chain, and the step `ipi-conf-gcp-private-dns-zone`, but not in the step `gcp-provision-private-dns-zone-in-host-project`?

Reply:
I meant `CLUSTER_PVTZ_PROJECT` can be removed from the beginning.